saleor/checkout/models.py

```diff
@@ -250,6 +247,30 @@ def get_country(self):
             self.set_country(country_code, commit=True)
         return country_code
 
```

Comment:
Not sure if setting `CheckoutMetadata` should be part of a `Checkout` model or rather a helper method.
Also I was wondering if proper permission checks are made in order to save `privateMetadata`